From 7eec3fabafa9f1145cafe96c5dba8356faf23c31 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Sun, 01 Mar 2020 08:26:34 +0200
Subject: [PATCH] [SW-8229] goya2: fix RR high bandwidth define

Define value wasn't set to correct range because shift needed to be by
12 bit.

Change-Id: I795b05b0d0b95289991b3a63299eb1c6f64b0eb3
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/common/security.h b/common/security.h
index 9a3c6ff..66e0ba7 100644
--- a/common/security.h
+++ b/common/security.h
@@ -76,11 +76,11 @@
 #define SECURITY_PSOC_START_HBW_MSB		0x0
 #define SECURITY_PSOC_START_HBW_LSB		0x0
 
-#define SECURITY_QSPI_START_HBW			(mmPSOC_QSPI_BASE >> 24)
-#define SECURITY_QSPI_END_HBW			(mmPSOC_UART_0_BASE >> 24)
-#define SECURITY_PSOC_SECURITY_START_HBW	(mmPSOC_SECURITY_BASE >> 24)
+#define SECURITY_QSPI_START_HBW			(mmPSOC_QSPI_BASE >> 12)
+#define SECURITY_QSPI_END_HBW			(mmPSOC_UART_0_BASE >> 12)
+#define SECURITY_PSOC_SECURITY_START_HBW	(mmPSOC_SECURITY_BASE >> 12)
 #define SECURITY_PSOC_SECURITY_END_HBW		(mmJT_MSTR_IF_RR_SHRD_HBW_BASE \
-						 >> 24)
+						 >> 12)
 #define SECURITY_FLASH_START_HBW		0x7FFA000
 #define SECURITY_FLASH_END_HBW			0x7FFB000
 
